salt/modules/mount.py

```diff
@@ -960,3 +960,30 @@ def is_mounted(name):
         return True
     else:
         return False
+
+
+def get_othermountpoints(to_ignore=[]):
```

Comment:
I understand where you're going with this function but to be honest, I think it's going to be way too hard to maintain this across multiple platforms and systems. There's really no way here to see what this function does without looking at the code itself. If we were to do something like this, my suggestion would be that we use the Filesystem Hierarchy Standard, though even then there are operating systems which don't comply...